appveyor.yml

```diff
@@ -15,8 +10,6 @@ environment:
     - nodejs_version: "6"
 
 install:
-  - npm -g install npm@latest
```

Comment:
My reply crossed with yours.

> Just install npm at a good version, like 2

I got it. I checked our test succeeded with `npm@2` now.